src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSource.java

```diff
@@ -113,7 +113,7 @@
     private String sshRemote;
     private String httpRemote;
     private transient Project gitlabProject;
-    private int projectId = -1;
+    private String projectId;
```

Comment:
```suggestion
    private int projectId;
```
This change can be achieved without changing it to a string.